pkg/cmd/admin/admin.go

```diff
@@ -27,15 +27,15 @@ import (
 	exipfailover "github.com/openshift/origin/pkg/cmd/experimental/ipfailover"
 	"github.com/openshift/origin/pkg/cmd/server/admin"
 	"github.com/openshift/origin/pkg/cmd/templates"
+	. "github.com/openshift/origin/pkg/cmd/templates"
```

Comment:
Sounds good, it's already a dependency and kube uses it in some cmds. I'll include heredoc to the normalization logic.